src/utils/enums/error-codes.ts

```diff
@@ -0,0 +1,4 @@
+export enum ErrorStatusCodes {
+  ACCOUNT_DELETED = 72,
+  INTERNAL_ERROR = -32603,
+}
```

Comment:
So there's `ErrorStatusCodes` and also `ERROR_STATUS_CODES`? Could these be combined in any way? Or is one of them unnecessary?

Reply:
Removed the `ERROR_STATUS_CODES`